### DISCUSSION TITLE
Table block rendering bugs

### DISCUSSION THREAD

**Discussion by gitacoco** [GitbookIO/community#803] [0. General bugs and questions]
Hi there,

I have encountered several rendering bugs of the table block.

In Chrome/Arc browser environments, the cell content displays unexpected line breaks (highlighted with red boxes) and paragraph gaps (indicated with blue arrows):
![WX20241117-014428@2x](https://github.com/user-attachments/assets/6f442f94-fb1d-4bdd-bc8a-22ccbfa1c729)

In the Safari environment, the issue is improved but still exists:
![WX20241117-014502@2x](https://github.com/user-attachments/assets/a29b06ac-68ab-499d-8fce-7fb56fed5c53)

In the Edge environment, the issue is improved but still exists:
<img width="1757" alt="image" src="https://github.com/user-attachments/assets/87dcd3cc-9663-461a-a8f6-e7ad872cd4bb">

The page url: https://hifa.zhibang.design/prep/performance

Please investigate this issue. Thank you!

**Comment by zenoachtig** [comment]
Hi @gitacoco, thanks for reaching out! I’ll put this on my list of inconsistencies between the editor and published content. 

It’s most likely something to do with the way text and forced line breaks work inside tables. Especially interesting to see that it’s so different across browsers, might that have something to do with the font size or rendering of Chinese characters? It seems the 60分 line fits neatly on one line in Safari but doesn’t in other browsers.

 I’ll have a look, thanks for the detailed report!
